bin/jupyter-dashboards-server

```diff
@@ -41,9 +42,12 @@ if (config.get('SSL_OPTIONS')) {
     }).listen(port, ip);
     debug('Server set to HTTPS mode');
 } else {
+    activePort = port;
     server = http.createServer(app).listen(port, ip);
     debug('Server set to HTTP mode');
 }
+// save the correct port for easy lookup in routes, etc.
+config.set('ACTIVE_PORT', activePort);
```

Comment:
This breaks in Docker containers among other places where the server port is mapped to an external port. The port has to be read from the request.